localstack/.drone.yml

```diff
@@ -0,0 +1,79 @@
+---
+kind: pipeline
+type: docker
+name: localstack
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
```

Comment:
can we use a health check similar to what we did in the merged mysql example rather than a sleep.

Reply:
Oh wow! Yeah thats a much better approach waiting for the service to become available.

i will definitely do so